e2e/components/ProductiveCard/ProductiveCard-test.avt.e2e.js

```diff
@@ -23,4 +24,226 @@ test.describe('ProductiveCard @avt', () => {
       'ProductiveCard @avt-default-state'
     );
   });
+
+  test('@avt-with-caption', async ({ page }) => {
+    await visitStory(page, {
+      component: 'ProductiveCard',
+      id: 'ibm-products-components-cards-productivecard--with-caption',
+      globals: {
+        carbonTheme: 'white',
+      },
+    });
+    await expect(page).toHaveNoACViolations('ProductiveCard @avt-with-caption');
+  });
+
+  // Disabled state test
+  test('@avt-disabled: validates disabled button state', async ({ page }) => {
+    await visitStory(page, {
+      component: 'ProductiveCard',
+      id: 'ibm-products-components-cards-productivecard--with-action-ghost-button',
+    });
+
+    await expect(page).toHaveNoACViolations('ProductiveCard @avt-disabled');
+
+    const buttons = page.locator(`button.${carbon.prefix}--btn`);
+    const disabledButton = buttons.nth(2);
+    expect(await disabledButton.getAttribute('disabled')).not.toBeNull();
+
+    await page.keyboard.press('Tab');
+    expect(
+      await buttons.nth(0).evaluate((btn) => document.activeElement === btn)
+    ).toBe(true);
+
+    await page.keyboard.press('Tab');
+    expect(
+      await buttons.nth(1).evaluate((btn) => document.activeElement === btn)
+    ).toBe(true);
+    // disabled button
+    await page.keyboard.press('Tab');
+    expect(
+      await buttons.nth(2).evaluate((btn) => document.activeElement === btn)
+    ).toBe(false);
+  });
+
+  // Overflow menu open/close states test
+  test('@avt-overflow-menu: validates overflow menu interactions', async ({
+    page,
+  }) => {
+    await visitStory(page, {
+      component: 'ProductiveCard',
+      id: 'ibm-products-components-cards-productivecard--with-overflow',
+    });
+
+    const menuButton = page.locator('button[aria-haspopup="true"]');
+    const menu = page.locator('ul[role="menu"]');
```

Comment:
We can probably use the roles here as well. :)

https://playwright.dev/docs/locators#locate-by-role